packages/autoloader/src/AutoloadProcessor.php

```diff
@@ -88,6 +88,7 @@ public function processClassmap( $autoloads, $scanPsrPackages ) {
 			}
 		}
 
+		// PSR-0 namespaces are converted to classmaps for both optimized and unoptimized autoloaders.
```

Comment:
Would it be worth briefly stating why? e.g. "because any new development should be using PSR-4 instead."

Reply:
Yes, good suggestion! I added that in commit 48fa31c. We actually use classmap autoloading in the Jetpack packages. [Here's the PR](https://github.com/Automattic/jetpack/pull/13913) where we switched from PSR-4 to classmap, and [here's a link](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#naming-conventions) to the naming conventions section of the WordPress coding standards.

In that commit, I also added some info about the autoloading standards to the Packages README file and the Autoloader README file.